app/core/tests/test_api.py

```diff
@@ -156,3 +156,17 @@ def test_create_sponsor_partner(auth_client):
     }
     res = auth_client.post(SPONSOR_PARTNERS_URL, payload)
     assert res.status_code == status.HTTP_201_CREATED
+
+
+def test_create_faq(admin_client):
+
+    payload = {
+        "question": "How do I work on an issue",
+        "answer": "See CONTRIBUTING.md",
+        "tool_tip_name": "How to work on an issue",
+    }
+    res = admin_client.post(FAQ_URL, payload)
+    assert res.status_code == status.HTTP_201_CREATED
```

Comment:
Extra: You can also test the response data if you want. 
like this or similar: `assert res.data["question"] == payload["question"]`
You can't assert equality on the dictionary because the API data includes the uuid field.

See [this](https://github.com/fyliu/django-tdd-docker/blob/294b216d7e0ba5841b064b63c3bdbbc3a147d645/app/tests/movies/test_views.py#L21) for reference.